pages/city/[city].tsx

```diff
@@ -0,0 +1,255 @@
+                <OverlayTrigger
+                  overlay={
+                    <Tooltip id="tooltip-disabled">
+                      <Image
+                        src={`/image/qrcode/${wechat}`}
+                        className="w-75 h-75"
+                        alt={name}
+                      />
+                    </Tooltip>
+                  }
+                  placement="right"
+                >
+                  <span className="d-inline-block text-success">微信</span>
+                </OverlayTrigger>
+              </Col>
+            )}
+            {weibo && (
+              <Col as="li" id="weiboLink" className="py-1">
+                <a
+                  href={weibo}
+                  target="_blank"
+                  className="text-success"
+                  rel="noreferrer"
+                >
+                  微博
+                </a>
+              </Col>
+            )}
+            {github && (
+              <Col as="li" id="githubLink" className="py-1">
+                <a
+                  href={github ? github : '/'}
+                  target="_blank"
+                  className="text-success"
+                  rel="noreferrer"
+                >
+                  GitHub
+                </a>
+              </Col>
+            )}
+          </Row>
+        </div>
+      </section>
+
+      {organisers?.[0] && (
+        <section className="text-center mx-auto my-0">
+          <h2 id="organisers" className="fs-4 m-0 py-5 text-start">
+            社区组织者
+          </h2>
+          <Row
+            as="ul"
+            className="list-unstyled text-center justify-content-around g-5"
+            xs={2}
+            sm={5}
+          >
+            {organisers.map(({ name, link, pic }) => (
+              <Col
+                as="li"
+                className={`${styles.media} mt-1 pt-5 px-1`}
+                key={pic}
+              >
+                {link ? (
+                  <a
+                    className="text-dark"
+                    href={link ? `/organizer/${link}` : ``}
+                  >
+                    <Image
+                      style={{ width: '9rem', height: '9rem' }}
+                      src={`/image/organizer/${pic}`}
+                      alt={name}
+                    />
+                    <ul className="list-unstyled mt-3">
+                      <li className="fs-6 fw-bolder mt-1">{name}</li>
+                    </ul>
+                  </a>
+                ) : (
+                  <>
+                    <Image
+                      style={{ width: '9rem', height: '9rem' }}
+                      src={`/image/organizer/${pic}`}
+                      alt={name}
+                    />
+                    <ul className="list-unstyled mt-3">
+                      <li className="fs-6 fw-bolder mt-1">{name}</li>
+                    </ul>
+                  </>
+                )}
+              </Col>
```

Comment:
```suggestion
              <Col
                as="li"
                className={`${styles.media} mt-1 pt-5 px-1 position-relative`}
                key={pic}
              >
                    <Image
                      style={{ width: '9rem', height: '9rem' }}
                      src={`/image/organizer/${pic}`}
                      alt={name}
                    />
                    <a
                      className="stretched-link text-dark fw-bolder mt-3"
                      href={link ? `/organizer/${link}` : '#'}
                    >
                      {name}
                    </a>
              </Col>
```
https://getbootstrap.com/docs/5.3/helpers/stretched-link/